fluent-plugin-kubernetes-sumologic/lib/fluent/plugin/filter_kubernetes_sumologic.rb

```diff
@@ -156,77 +134,43 @@ def filter(tag, time, record)
           return nil
         end
 
-        sumo_metadata[:log_format] = annotations["sumologic.com/format"] if annotations["sumologic.com/format"]
-
-        if annotations["sumologic.com/sourceHost"].nil?
-          sumo_metadata[:host] = sumo_metadata[:host] % k8s_metadata
-        else
-          sumo_metadata[:host] = annotations["sumologic.com/sourceHost"] % k8s_metadata
```

Comment:
good question - for default source name, we use something like `%{namespace}.%{pod}.%{container}`, so if `k8s_metadata` contains keys for `namespace`, `pod`, or `container`, it will populate those values so we get a final source name like `sumologic.fluentd-xxx.fluentd` or something like that

I believe this was how we supported dynamically populating metadata values in our sumo metadata.